docker/api/image.py

```diff
@@ -377,7 +377,11 @@ def pull(self, repository, tag=None, stream=False, auth_config=None,
 
         Example:
 
-            >>> for line in client.api.pull('busybox', stream=True, decode=True):
+            >>> for line in client.api.pull(
+                'busybox',
+                stream=True,
+                decode=True
+            ):
```

Comment:
This should probably look like this according to other places:
```suggestion
                decode=True):
```